docs/index.bs

```diff
@@ -809,8 +809,9 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
         1. Let |promise| be a new <a>promise</a>.
         1. Run the following substeps <a>in parallel</a>:
             1. Let |registration| be the result of running <a>Match Service Worker Registration</a> given |storage key| and |clientURL|.
```

Comment:
I am trying to understand the algorithm and just a question but, why you want to run "Let |registration| ..." step in parallel?
If it is within a queue, once |registration| is not null, it should be kept on non-null.  However, the current algorithm put it outside, during race condition, |promise| may be resolved undefined even after |promise| is resolved with the result.

Reply:
Yes you are right, it should be in a queue. Updated!